src/Console/Command/CommandHelp.php

```diff
@@ -298,9 +353,53 @@ function (FixerInterface $a, FixerInterface $b) {
                 $help .= sprintf(" * <comment>%s</comment>\n   | %s\n", $fixer->getName(), $description);
             }
 
-            if (count($attributes)) {
-                sort($attributes);
-                $help .= sprintf("   | *Rule is: %s.*\n", implode(', ', $attributes));
+            if ($fixer->isRisky()) {
+                $help .= "   | *Risky rule.*\n";
```

Comment:
maybe replace it with sth like `Rule may be risky: $descriptionFromDefinition` ?